benchmarks/perfetto/framework.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+from .backends import AVAILABLE_BACKENDS, DEFAULT_METRICS
+from .table import TraceAnalysisBenchmarkResult
+
+
+class TraceAnalysisBenchmark:
+    def __init__(self, args: argparse.Namespace):
+        self.inputs = args.inputs
+        self.tasks = args.tasks
+        self.backends = set(map(lambda x: x(args), filter(lambda x: x, map(lambda x: AVAILABLE_BACKENDS.get(x, None), args.backends))))
```

Comment:
Nit: Perhaps we can add some line breaks and indentation to make this a bit easier to read.